sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_exporter.py

```diff
@@ -348,7 +350,7 @@ def _convert_span_to_envelope(span: ReadableSpan) -> TelemetryItem:
                 # data
                 if url:
                     data.data = url
-                target, path = trace_utils._get_target_and_path_for_http_dependency(
+                target, path = trace_utils._get_target_and_path_for_http_dependency_with_gen_ai(
```

Comment:
You don't need to change any other logic. Simply just put the `if gen_ai_attributes.GEN_AI_SYSTEM in span.attributes:  # GenAI` at the top before `if _AZURE_SDK_NAMESPACE_NAME in span.attributes:` and then `_get_target_for_http_dependency_with_gen_ai`. (no need for path).

Reply:
changed in another way. remain utils untouched. and if target is not set, then set it to gen_ai directly.